_posts/2020-12-10-share-your-knowledge.markdown

```diff
@@ -0,0 +1,59 @@
+
+As time went on, other product teams started their own Knowledge Shares. They proliferated naturally throughout Artsy Engineering. When I started the Mobile Experience team in 2019, Knowledge Shares were one of the first recurring team meetings I scheduled. After a few months, I got really positive feedback about them. In fact, engineers wanted more of them. So I scheduled a second hour-long Knowledge Shares each week.
+
+Today, Knowledge Shares are for more than just engineers. They are for product managers, designers, data analysts, and other team stakeholders. Sometimes engineers from other teams even join my team's Knowledge Shares, but scheduling becomes a hassle at a certain scale. We start each Knowledge Share with team-wide topics that apply to more than only engineers; we then move on to engineering-specific topics. This lets non-engineers still contribute while also letting them drop off so the engineers can dig into code.
+
+Here are some examples of things that we have used Knowledge Shares for:
+
+- Sharing early designs for upcoming project work.
+- Going over work-in-progress pull requests and soliciting feedback on the technical approach.
+- Spiking on big-picture projects together, to help us plan what work needs to happen and in which order.
+- Exploring anonymized user sessions to learn more about how our product actually gets used in the real world.
+- Investigating small quality-of-life problems with our codebase's developer experience and working on solutions.
+- Brainstorming about how to address specific pieces of technical debt.
+
+Topics range from the technical to the product-focused, from what has already shipped to what is still on the drawing board. This breadth of focus helps everyone on the team feel like they can contribute. We keep a shared document of evergreen topic ideas and everyone is encouraged to add to it. As Tech Lead, I also send Slack reminders before each meeting to solicit for timely topics.
+
+Throughout the normal course of our week, topics will naturally come up for discussion – topics that would normally need their own meeting to be scheduled. But instead, we can use the time we already have scheduled. For example: if an engineer has feedback on a new feature's design, we don't need to schedule a dedicated meeting for that conversation – we can use a Knowledge Share. It might feel like having recurring "Knowledge Share" meetings only adds more meetings to everyone's already-busy calendars. But in our experience, these Knowledge Shares prevent us from scheduling _yet more_ one-off meetings, so it balances out as a time-saver
```

Comment:
```suggestion
Throughout the normal course of our week, topics will naturally come up for discussion – topics that would normally need their own meeting to be scheduled. But instead, we can use the time we already have scheduled. For example: if an engineer has feedback on a new feature's design, we don't need to schedule a dedicated meeting for that conversation – we can use a Knowledge Share. It might feel like having recurring "Knowledge Share" meetings only adds more meetings to everyone's already-busy calendars. But in our experience, these Knowledge Shares prevent us from scheduling _yet more_ one-off meetings, so it balances out as a time-saver.
```

Adding a period, also I *love* this point about preventing us from scheduling *more* meetings.